app/templates/mandaat/edit.hbs

```diff
@@ -1,3 +1,24 @@
-{{page-title "Edit"}}
-<Tmp::Todo>Allow edit of mandaat</Tmp::Todo>
+{{page-title "Edit Mandaat"}}
+<AuToolbar @border="bottom" @size="large" as |Group|>
+  <Group>
+    <AuHeading @skin="2">Bewerk Mandaat
+      {{this.model.mandaat.bestuursfunctie.label}}</AuHeading>
+  </Group>
+</AuToolbar>
+
+<div class="au-c-body-container au-c-body-container--scroll">
+  <div class="au-o-box">
+    <Form::Instance
+      @instanceId={{this.model.instanceId}}
+      @form={{this.model.form}}
+      @onSave={{this.onSave}}
+    />
+  </div>
+</div>
 {{outlet}}
+
+<AuToolbar @border="top" @size="large" as |Group|>
```

Comment:
Might be nice to just give this onCancel action to the form instance, than you have the two buttons next to each other. But depends on preference of course.

Reply:
fair point!